phpstan.neon

```diff
@@ -0,0 +1,4 @@
+parameters:
+    ignoreErrors:
+        # phpstan seems to want to trust to the phpdoc?
+        - '#Result of || is always false#'
```

Comment:
am i missing something or is phpstan saying we should not verify that the user of our code respects the phpdoc?

Reply:
Seems like a wrong ignore here. Wherever this was ignored, it more than likely points to some bug potential. Might also be that the phpdoc type is not explicit enough and then this gets triggered, so maybe an improvement needs to be made to be more clear and explicit around that area.

Reply:
reported the problem to phpstan: https://github.com/phpstan/phpstan/issues/1926